data/talkactions/scripts/start_cast.lua

```diff
@@ -0,0 +1,9 @@
+function onSay(cid, words, param)
+	local player = Player(cid)
```

Comment:
But if you ask me you should go with just one script, use just !cast or & /cast to load the script, then use something like Player:setCastState(bool) insted of startLiveCast / stopLiveCast.
Problem will be the password, but you can just as easy check if the player even has any parameters to giev and in that case use Player:setCastPassword ~